app/models/job/Worker.scala

```diff
@@ -81,6 +83,12 @@ class WorkerService @Inject()(conf: WkConf) {
       "lastHeartBeatIsRecent" -> lastHeartBeatIsRecent(worker)
     )
 
+  def assertDataStoreHasWorkers(dataStoreName: String)(implicit ctx: DBAccessContext): Fox[Boolean] =
+    for {
+      _ <- dataStoreDAO.findOneByName(dataStoreName)
+      _ <- workerDAO.findOneByDataStore(dataStoreName)
```

Comment:
```suggestion
      _ <- workerDAO.findOneByDataStore(dataStoreName) ?~> "jobs.noWorkerForDatastore"
```
If adding the messages happens here, it can be removed from the controller, saving quite a few redundant strings :)
In the mesesages file, I’d suggest `Processing jobs are not available for the datastore this dataset belongs to`